src/xmipp/libraries/data/normalize.cpp

```diff
@@ -251,7 +251,7 @@ void normalize_Robust(MultidimArray<double> &I, const MultidimArray<int> &bg_mas
     {
         Image<double> mask;
         mask() = I;
-        double th = EntropySegmentation(mask());
+        static_cast<void>(EntropySegmentation(mask()));
```

Comment:
why do you do the cast?

Reply:
Entropy segmentation function already retrieves the masked map through reference. However, it also returns the threshold needed to get that mask from the volume through the "return" of the function. Since I only need the mask and not the threshold, there is a cast to void to tell the compiler that we do not care about the return of the function (otherwise, it value will be unused leading to a code smell in sonar cloud).